src/Services/Reducers/index.js

```diff
@@ -0,0 +1,15 @@
+import { toCamelCase } from '../Utils';
+import { FETCH_VIDEOS } from '../Constants';
+
+const defaultState = {
+  videos: [],
+};
+
+export default (state = defaultState, action) => {
+  switch (action.type) {
+    case FETCH_VIDEOS:
+      return { ...state, videos: [...toCamelCase(action.payload.data)] };
```

Comment:
no need to do new array generation for toCamelCase(action.payload.data) here, as it generates new array inside alread, so you can just 
```
return { ...state, videos: toCamelCase(action.payload.data) };
```